x-pack/solutions/observability/plugins/slo/server/services/create_slo.ts

```diff
@@ -99,11 +97,7 @@ export class CreateSLO {
         tempDocPromise,
       ]);
 
-      rollbackOperations.push(() => this.transformManager.stop(rollupTransformId));
-      rollbackOperations.push(() => this.summaryTransformManager.stop(summaryTransformId));
-
```

Comment:
stopping the transform is not necessary anymore since uninstalling will proceed regardless of the transform state